src/pages/[[...markdownPath]].js

```diff
@@ -89,12 +89,15 @@ function reviveNodeOnClient(key, val) {
       type = Fragment;
     }
     return {
-      $$typeof: Symbol.for('react.element'),
+      $$typeof: Symbol.for('react.transitional.element'),
       type: type,
       key: key,
       ref: null,
       props: props,
       _owner: null,
+      _store: {
+        validate: false,
+      },
```

Comment:
https://github.com/facebook/react/blob/c56c6234328a29930487295afe61597db48f058c/packages/react/src/ReactChildren.js#L242-L250

Im unaware of what this piece of code does so Im currently adding _store to be `null`. Should I update react to soft check if `_store` is present or not?